src/mapml/layers/MapLayer.js

```diff
@@ -196,35 +301,29 @@ export var MapMLLayer = L.Layer.extend({
         map.on("popupopen", this._attachSkipButtons, this);
     },
 
+
     _validProjection : function(map){
       let noLayer = false;
-      if(this._templateVars){
-        for(let template of this._templateVars)
-          if(!template.projectionMatch && template.projection !== map.options.projection) noLayer = true;
+      if(this._extent && this._extent._mapExtents){
+        for(let i = 0; i < this._extent._mapExtents.length; i++){
+          if(this._extent._mapExtents[i]._templateVars){
+            for(let template of this._extent._mapExtents[i]._templateVars)
+              if(!template.projectionMatch && template.projection !== map.options.projection) {
+                noLayer = true; // if there's a single template where projections don't match, set noLayer to true
+                break;
+              } 
+            }
       }
+    }
       return !(noLayer || this.getProjection() !== map.options.projection.toUpperCase());
     },
 
     //sets the <layer-> elements .bounds property 
     _setLayerElExtent: function(){
```

Comment:
Probably need to get rid of this method entirely, keep (a possibly renamed)_getCombinedExtentsLayerBounds (above, L113).